public/app/features/browse-dashboards/api/browseDashboardsAPI.ts

```diff
@@ -431,6 +431,7 @@ export const {
   useDeleteItemsMutation,
   useGetAffectedItemsQuery,
   useGetFolderQuery,
+  useLazyGetFolderQuery,
```

Comment:
@grafana/grafana-frontend-platform any particular reason why we're only exporting a limited set of hooks from the API endpoints? 🤔

Reply:
we've just been exporting what we use